cmd/devices-api/main.go

```diff
@@ -91,7 +91,6 @@ func main() {
 		subcommands.Register(&remakeUserDeviceTokenTableCmd{logger: logger, settings: settings, pdb: pdb, container: deps}, "device integrations")
 		subcommands.Register(&remakeFenceTopicCmd{logger: logger, settings: settings, pdb: pdb}, "device integrations")
 		subcommands.Register(&remakeDeviceDefinitionTopicsCmd{logger: logger, settings: settings, pdb: pdb, ddSvc: deps.getDeviceDefinitionService()}, "device integrations")
-		subcommands.Register(&startSmartcarFromRefreshCmd{logger: logger, settings: settings, pdb: pdb, ddSvc: deps.getDeviceDefinitionService()}, "device integrations")
```

Comment:
I assume this is just refactoring not related to the current story backed by PR?

Reply:
No, this is a very old task that we don't use anymore—Smartcar doesn't give us refresh tokens anymore if things go wrong—and I don't want to upgrade it to use this stuff. I think it's kinda related.